tests/conftest.py

```diff
@@ -32,8 +30,6 @@ def test_while_anon(clinet):
 
 @pytest.fixture(scope='module')
 def app():
-    os.environ['AIRFLOW__CORE__SQL_ALCHEMY_CONN'] = 'sqlite:///testdb.sqlite'
```

Comment:
Why did remove this?

Reply:
it's requesting for full path instead of a relative path and I decided to use the default